### PR TITLE
Feature to start process in camunda

### PR DESCRIPTION
**Pull request by Waldz**

### CODE REVIEW

File: camunda_example/main.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	httpTransport := &http.Transport{
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true,
+		},
+	}
+	httpClient := http.Client{Transport: httpTransport}
+
+	camunda := rest.Construct("https://localhost:6002/engine-rest", "admin", "admin", httpClient)
+	camunda.HandleErrors(func(err error) {
+		fmt.Printf("\nError: %#v", err)
```
Comment (ghost) [inline camunda_example/main.go:21]: Ugly. Try err.Error()

File: camunda_example/main.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+func main() {
+	httpTransport := &http.Transport{
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true,
+		},
+	}
+	httpClient := http.Client{Transport: httpTransport}
+
+	camunda := rest.Construct("https://localhost:6002/engine-rest", "admin", "admin", httpClient)
+	camunda.HandleErrors(func(err error) {
```
Comment (ghost) [inline camunda_example/main.go:20]: Overenginering IMO. All errors are returned anyway

**Comment by anroots** [comment]
Missing copyright holder name in license file: https://github.com/bigbank-as/go-camunda-client/blob/master/LICENSE#L189

First @bigbank-as Go PR, (y)!

File: client_interface.go [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package camunda_client
+
+type CamundaClient interface {
+	StartProcess(processDefinitionKey string, request ProcessStartRequest) (Process, error)
+	GetProcess(processId string) (Process, error)
+	HandleErrors(errorCallback func(error))
+}
+
+type ProcessStartRequest interface{}
```
Comment (ghost) [inline client_interface.go:9]: I don't see a point in this. Just use interface{} where it's needed

File: rest/dto/error.go [hunk 4]
```diff
@@ -0,0 +1,6 @@
+package dto
+
+type Error struct {
```
Comment (ghost) [inline rest/dto/error.go:3]: Is it used anywhere?

File: camunda_example/main.go (hunk 1 above)
Comment (Waldz) [reply camunda_example/main.go:21]: Fixed

File: camunda_example/main.go (hunk 2 above)
Comment (Waldz) [reply camunda_example/main.go:20]: This will become request/response logger to file/db

File: rest/dto/error.go (hunk 4 above)
Comment (Waldz) [reply rest/dto/error.go:3]: Yes, Dto of API remote errors

**Pull request merged**